packages/eslint-plugin-pf-codemods/src/rules/v6/loginMainFooterLinksItemStructureUpdated/loginMainFooterLinksItemStructureUpdatedOutput.tsx

```diff
@@ -0,0 +1,14 @@
+import { LoginMainFooterLinksItem } from "@patternfly/react-core";
```

Comment:
The fix command just needs to be ran to update the output so that Button gets imported